### DISCUSSION TITLE
Spacy Resume/CV parser

### DISCUSSION THREAD

**Discussion by Dinushan-S** [explosion/spaCy#13258] [New Features & Project Ideas]
Hello everyone,

I am currently working on a project where I need to extract key information such as names, numbers, and skills from resumes. These resumes are initially in PDF or DOC format, which I convert into text before processing them with Spacy.

However, I am facing challenges in achieving high accuracy in the extraction process. I was wondering if anyone here has experience with similar tasks and could share their insights or even point me towards any repositories that might be helpful.

Any guidance or suggestions would be greatly appreciated. Thank you!

**Comment by svlandeg** [comment]
Have you seen [this detailed blog post](https://deepnote.com/@abid/spaCy-Resume-Analysis-81ba1e4b-7fa8-45fe-ac7a-0b7bf3da7826) by Abid Ali Awan on resume parsing?

**Comment by Dinushan-S** [comment]
i try that but that not working
